docs/reference/commandline/build.md

```diff
@@ -345,6 +345,13 @@ This will use a file called `Dockerfile.debug` for the build instructions
 instead of `Dockerfile`.
 
 ```bash
+$ docker build -f - . < ../Dockerfile
+```
+
+The above command will use the current directory as the build context using
+a Dockerfile from stdin.
```

Comment:
Should we explain what the advantage is of this? i.e., when building from `stdin`, you can use a Dockerfile that's outside of the build-context?